db/queries.py

```diff
@@ -3,44 +3,67 @@
 
 
 def get_messages_that_contain_word(word: str) -> list[Message]:
-    pass
+    return [message for message in Message.objects.filter(text__contains=word)]
```

Comment:
It is better to use list() method for converting QuerySet to the list, cause u'll write less code